tic_tac_toe/game.py

```diff
@@ -33,7 +43,14 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    
```

Comment:
Just a note, you could have just iterated through the board instead of using range here:
```
for row in board:
    for cell in row:
        if cell == '-':
            return False
return True
```
This would be more functional and doesn't rely on the board being a specific size.